website_portal_contract/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================
+Website Portal Contract
+=======================
+
+Extension of contract and website_portal that allows contracts to
+be shown in website portal. Each contract now has a website template tied to it.
+
+To edit the website template, go to:
+
+* `Invoicing` in the top navigation.
+* `Contracts` => `Website Templates`.
+* When viewing a template in form view, click `View Template`.
+  This will direct you to an edit page of the template in website if
+  your user has edit permissions.
+
+To view the live template in `My Account` in website, assign the template
+to a contract and go to:
+
+* `Website`
+* Click on your username then select `My Account` in the dropdown.
+* Click `Your Contracts`, then the relevant contract. Your template will show under
+  the `Contract` section header, which is also shown in the edit template view for reference.
+
+Usage
+=====
+
```

Comment:
😄 nice https://youtu.be/PSulRVdh4C4

Reply:
Welp now we have an instructional video! Let's add a link to this, and move the usage stuff from above (starting at "To edit the website template") to this section